### PR TITLE
readd group member property

### PR DESCRIPTION
**Pull request by jaxxstorm**

### CODE REVIEW

**Comment by jaxxstorm** [comment]
Fixes #19

File: provider/resources.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func Provider() tfbridge.ProviderInfo {
 		GitHubOrg:  "databricks",
 		Config:     map[string]*tfbridge.SchemaInfo{},
 		IgnoreMappings: []string{
-			"databricks_group_member", // causes the schema to have no deterministic generation
+			// "databricks_group_member", // causes the schema to have no deterministic generation
```
Comment (guineveresaenger) [inline provider/resources.go:70]: did you mean to leave this in?

**Review by guineveresaenger — COMMENTED** [review]
We don't use the CHANGELOG to write changes anymore - could you write the changelog line to the PR title instead and include a description?

As well, have we fixed the nondeterministic schema root cause?

**Comment by viveklak** [comment]
Superseded by #38 (this is a forked PR so easier to extend in a separate PR). I dropped the problematic docs for now for group_member resource. I will open a separate issue to fix example translation for the HCL examples for this resource.

**Pull request merged**